datasets/pib/README.md

```diff
@@ -0,0 +1,39 @@
+---
+YAML tags:
+- copy-paste the tags obtained with the tagging app: http://192.168.1.5:8501
```

Comment:
Could you fill the tags of the dataset card please ?
see https://github.com/huggingface/datasets/blob/master/ADD_NEW_DATASET.md#tag-the-dataset-and-write-the-dataset-card for more info